### PR TITLE
Miscellaneous cleanups

### PR DESCRIPTION
**Pull request by Kerollmops**

### CODE REVIEW

**Comment by koivunej** [comment]
Sadly this one conflicts as well.

File: src/lib.rs [hunk 1]
```diff
@@ -232,9 +230,9 @@ pub enum ContentType {
 
 impl Copy for ContentType {}
 
-impl Into<i32> for ContentType {
-    fn into(self) -> i32 {
-        match self {
+impl From<ContentType> for i32 {
```
Comment (koivunej) [inline src/lib.rs:233]: I did not know one could implement From my types for types in other crates. Interesting. I wonder why there is no lint for implementing Into then...?

**Comment by koivunej** [comment]
Thanks :+1: The comment I posted is only a question, in case you'd happen to know the answer :)

File: src/lib.rs (hunk 1 above)
Comment (Kerollmops) [reply src/lib.rs:233]: As the documentation say, this is not the right way to impl a convert trait, implementing the `From` trait is better than the [`Into`](https://doc.rust-lang.org/std/convert/trait.Into.html) one. (if I understand your question correctly 😄)

Comment (Kerollmops) [reply src/lib.rs:233]: I don't know why there is no warning from the compiler...

**Pull request merged**